main.go

```diff
@@ -164,55 +179,88 @@ func mainE(config Configs) error {
 	}
 
 	fmt.Println()
+	log.Infof("Export Artifacts:")
 
-	log.Infof("Export APKs:")
-	fmt.Println()
-
-	apks, err := getArtifacts(gradleProject, started, config.APKPathPattern, false)
+	artifacts, err := getArtifacts(gradleProject, started, appPatterns, false)
 	if err != nil {
-		return fmt.Errorf("failed to find apks, error: %v", err)
+		return fmt.Errorf("failed to find artifacts, error: %v", err)
+	}
+
+	var artNames []string
+	for _, a := range artifacts {
+		artNames = append(artNames, a.Name)
 	}
 
-	if len(apks) == 0 {
-		log.Warnf("No apks found with pattern: %s", config.APKPathPattern)
-		log.Warnf("If you have changed default APK export path in your gradle files then you might need to change APKPathPattern accordingly.")
+	log.Donef("Found artifacts by provided pattern:")
+
+	// Filter artifacts by build type
+	var filteredArtifacts []gradle.Artifact
+	for _, a := range artifacts {
+		if filepath.Ext(a.Path) == "."+config.BuildType {
```

Comment:
I think the log about the extension is not really meaningful, especially for our users. As it is visible from the file name also. The current log doesn't really contains useful information that will answer the users question if they don't find something. Please make sure that the user know that which pattern caused which artifact to be exported.

Reply:
Fixed in:
https://github.com/bitrise-steplib/bitrise-step-android-build/pull/11/commits/be0e44ca3dcd772e7831ba22ab20992a50453baa

<img width="765" alt="Screenshot 2019-05-24 at 12 48 24" src="https://user-images.githubusercontent.com/37296013/58322782-413cc400-7e22-11e9-816d-06e5c85a86da.png">